### PR TITLE
Move user data to portfolio page

### PR DESCRIPTION
**Pull request by sssmi**
# Description

- Move user investments data to portfolio page
- Add portfolio link to navigation
- Show Hero banners on portfolio and home page

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

Navigate to portfolio page and connect your wallet. You should see your investments there.

## Visual context

| Portfolio Page Not Connected | Portfolio Page Connected |
| ---------------------------- | ------------------------ |
|   Screen Shot 2022-06-09 at 9 02 00 |  Screen Shot 2022-06-09 at 9 01 47  |

| Invest Page Not Connected | Invest Page Connected |
| ---------------------------- | ------------------------ |
|  Screen Shot 2022-06-08 at 8 34 00 |  Screen Shot 2022-06-08 at 8 34 29 |

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/pages/_layouts/DefaultLayout.vue [hunk 1]
```diff
@@ -16,15 +16,19 @@ const { isDesktop } = useBreakpoints();
...
   <div>
     <AppNav />
-    <template v-if="isHomePage">
+    <template v-if="isHomePage || isPortfolioPage">
       <StakingProvider>
-        <AppHero />
+        <AppHero
+          :showInvestments="isPortfolioPage"
+          :showLearnMoreBtn="isHomePage"
+        />
```
Comment (garethfuller) [inline src/pages/_layouts/DefaultLayout.vue:31]: AppHero was initially used on every page, hence why we named it AppHero. I think we now need to split it into two separate components and use them on each individual page. e.g. on the home page have a HomePageHero and on the portfolio page PortfolioPageHero, or something similar to this.

Two reasons for this:
1. Nesting inside StakingProvider above means all the staking data is loaded on the home page where it's not needed.
2. There are queries being run in AppHero that are not needed on the home page so again there is excessive fetching.

**Review by garethfuller — CHANGES_REQUESTED** [review]
Looks good just one major comment about splitting up the AppHero component. It looks like we might still be fetching all the data on the home page that we only need to fetch on the portfolio page.

File: src/pages/_layouts/DefaultLayout.vue (hunk 1 above)
Comment (sssmi) [reply src/pages/_layouts/DefaultLayout.vue:31]: Makes sense. I will split the Hero components.

Comment (sssmi) [reply src/pages/_layouts/DefaultLayout.vue]: Alright. I've created PortfolioPageHero and HomePageHero components. AppHero is now composable component.

**Review by garethfuller — DISMISSED** [review]
It's looking great now! Thanks for making those changes. I just have one final comment.

File: src/pages/_layouts/DefaultLayout.vue [hunk 2]
```diff
@@ -16,17 +17,18 @@ const { isDesktop } = useBreakpoints();
...
-    <template v-if="isHomePage">
-      <StakingProvider>
-        <AppHero />
-      </StakingProvider>
-    </template>
+    <HomePageHero v-if="isHomePage" />
+    <StakingProvider v-else-if="isPortfolioPage">
+      <PortfolioPageHero :showInvestments="true" />
+    </StakingProvider>
+
```
[the remark this answers is not in the document]
Comment (sssmi) [reply src/pages/_layouts/DefaultLayout.vue]: Right. I've moved them now.

**Review by timjrobinson — APPROVED** [review]

**Comment by pkattera** [comment]
This works as expected for me!

**Review by pkattera — APPROVED** [review]
Works as expected for me

**Review by sssmi — COMMENTED** [review]
All done

**Pull request merged**